backend/src/middleware/permissionsMiddleware.js

```diff
@@ -7,7 +10,7 @@ import { rule, shield, deny, allow, and, or, not } from 'graphql-shield'
 const isAuthenticated = rule({
   cache: 'contextual',
 })(async (_parent, _args, ctx, _info) => {
-  return ctx.user !== null
+  return ctx.user != null
```

Comment:
Why not
```suggestion
  return ctx && ctx.user && ctx.user.id
```
?

Reply:
Sorry, I forgot to ask you about this! When we talked you were wondering how to test for `undefined` or `null` – I found out! :wink: But of course we can check for the user id if you think it makes more sense.